### ISSUE TITLE
different layouts for different keyboards

### ISSUE THREAD

**Issue by rjshrjndrn** [rvaiya/keyd#806]
Is it possible to load different layout for different keyboards?

For example, when I use my laptop keyboard I want to use a configuration, and when I use my split keyboard, I want to use a different layout.

**Comment by nsbgn** [comment]
Yes, you can put multiple configurations in `/etc/keyd/` with different IDs in the `[ids]` section. Each config will only apply to keyboards with a matching ID.

**Comment by rjshrjndrn** [comment]
Ah man.. Didn't think of that. thank you @nsbgn

**Comment by rjshrjndrn** [comment]
I don't think that's working, I don't know how the configurations are handled.

**Comment by tkna91** [comment]
It is documented. It may be difficult, but I encourage you to read it.
https://github.com/rvaiya/keyd/blob/master/docs/keyd.scdoc#L78
